src/services/fungibleTokens.js

```diff
@@ -0,0 +1,90 @@
+import * as Sentry from '@sentry/browser';
+import BN from 'bn.js';
+import * as nearApiJs from 'near-api-js';
+
+import sendJson from '../tmp_fetch_send_json';
+import { wallet } from '../utils/wallet';
+import { ACCOUNT_HELPER_URL } from '../utils/wallet';
+
+const FT_MINIMUM_STORAGE_BALANCE = '1250000000000000000000';
+const FT_STORAGE_DEPOSIT_GAS = '30000000000000';
+const FT_TRANSFER_GAS = '30000000000000';
+const FT_TRANSFER_DEPOSIT = '1';
+
+const {
+    transactions: {
+        functionCall
+    }
+} = nearApiJs;
+
+class FungibleTokens {
+    constructor() {
+        this.account = wallet.getAccountBasic();
```

Comment:
Right, but I'm not sure about stashing the instance in Redux because of redux-persist. Maybe this type of solution:
https://github.com/near/near-wallet/pull/1892/commits/2ce24bb12e03c82f168f2e2dfea9bc97710a3567

It would make sense to keep everything in wallet instance, and it would look much better codewise after we refactor wallet.js into a more reasonable form.

Reply:
Sure, we can keep it as a property of the wallet instance; makes sense to me :+1: